chef/data_bags/crowbar/template-nova.json

```diff
@@ -93,7 +97,7 @@
     "nova": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 43,
+      "schema-revision": 45,
```

Comment:
One commit moves to 44, and the next one to 45. We try to do revision bumps in an atomic way in terms of commits, so that any checkout would work.